### DISCUSSION TITLE
[Question] How do I change the selected "Data File"?

### DISCUSSION THREAD

**Discussion by TxTechnician** [2fast-team/2fast#46] [Q&A]
Is there a way to change the "Data File"?  

I cannot find an option to change the location.  And the text fields are not editable.  I'm using Windows 11 BTW.  

Screenshot:  
![image](https://user-images.githubusercontent.com/113131384/211401296-c71348de-ad82-42a2-8085-732deb849f16.png)

**Comment by jp-weber** [comment]
@TxTechnician 
If you want to permanently change the default file, you can reset the app to factory settings under options and functions. Alternatively, you can start another file by double-clicking on the file.

**Comment by TxTechnician** [comment]
Cool thanks.  I just made a copy of the file.  Would be nice to have the option to switch datafiles within the app, or even have the ability to create a hierarchy of some kind.  (want to keep my customers and multiple entities separate)

No complaints though.  Love your work thanks.

Andrew Valenzuela
Texas Technician
Phone:  806-560-0704
Email:  ***@***.***
Website:  www.txtechnician.com
